app/bundles/PageBundle/Form/Type/VariantType.php

```diff
@@ -70,13 +72,14 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 ],
                 'expanded'    => false,
                 'multiple'    => false,
-                'choices'     => $choices,
+                'choices'     => array_flip($choices),
```

Comment:
If I go to a page detail page and click on Clone from the contect menu I get

`500 Internal Server Error - PHP Warning - array_flip(): Can only flip STRING and INTEGER values! at /app/bundles/PageBundle/Form/Type/VariantType.php:75`

But I'm wondering whether it's not the same issue I had to deal with in the EmailBundle: 

https://github.com/mautic/mautic/pull/8049/files#diff-1675197cc33bd8f88473a64ff57d9e8cR146

Maybe it's solved for you too. Just remove the array_flip function. Please test it afterwards.

Reply:
Removed `array_flip` and it works. Input array had correct format.